babel.config.js

```diff
@@ -27,7 +26,8 @@ module.exports = (api) => {
 
   const buildPlugins = clean([
     '@babel/plugin-proposal-class-properties',
-    !isWebpack && '@babel/plugin-transform-react-constant-elements',
+    (isCJS || isES || isUMD || isRollup) &&
+      '@babel/plugin-transform-react-constant-elements',
```

Comment:
This seems to cover all conditions. When is this plugin skipped?

Reply:
it's skipped for webpack and parcel :)